plugins/org.eclipse.sirius.tests.swtbot/src/org/eclipse/sirius/tests/swtbot/modelexplorer/LockedModelExplorerTest.java

```diff
@@ -129,7 +130,17 @@ public void testRefreshJobForModelExplorerView() {
 
         assertFalse("The job should not be scheduled as no notification has been send.", refreshJobScheduled);
         lockRepresentation(true);
-        assertTrue("The job should be scheduled as one lock notification has been send and ModelExplorer view is opened.", refreshJobScheduled);
+        bot.waitUntil(new DefaultCondition() {
+            @Override
+            public boolean test() throws Exception {
+                return refreshJobScheduled;
```

Comment:
`refreshJobScheduled` should be either `volatile` or an `AtomicBoolean` if we want to see here the value which may have been updated from another thread.

Reply:
Done